src/Symfony/Component/AssetMapper/ImportMap/ImportMapConfigReader.php

```diff
@@ -23,11 +24,13 @@
 class ImportMapConfigReader
 {
     private ImportMapEntries $rootImportMapEntries;
+    private readonly Filesystem $filesystem;
 
     public function __construct(
         private readonly string $importMapConfigPath,
         private readonly RemotePackageStorage $remotePackageStorage,
     ) {
+        $this->filesystem = new Filesystem();
```

Comment:
framework bundle defines filesystem as a service, maybe inject it?

Reply:
Not sure if the cost/benefit ratio is worth it...  but not a strong opinion :) 

Maybe @derrabus (author of 54173) or @OskarStark (who suggested this PR in 54724) could help us decide ?

Reply:
As far as I can tell, we rarely really inject the `filesystem` service. I think, this is fine.